test/suite/shellEscape.test.ts

```diff
@@ -0,0 +1,30 @@
+import * as assert from 'assert';
+import * as shellEscape from '../../src/shellEscape';
+import * as sinon from 'sinon';
+import * as utils from '../../src/utils';
+
+
+suite('shellEscape', () => {
+  let sandbox: sinon.SinonSandbox;
+
+  setup(() => {
+    sandbox = sinon.createSandbox();
+  });
+
+  teardown(() => {
+    sandbox.restore();
+  });
+
+  suite('shellEscape', () => {
```

Comment:
To be thorough, can you also include tests for strings with quotes inside of them?